lib/addDefaults.js

```diff
@@ -19,6 +20,11 @@ function accessorDefaults(gltf) {
         if (accessors.hasOwnProperty(name)) {
             var accessor = accessors[name];
             accessor.byteStride = defaultValue(accessor.byteStride, 0);
+            if (!defined(accessor.min) || !defined(accessor.max)) {
```

Comment:
Soon, we will separate this into a new stage that adds min/max defaults.  We will call it here and also from a new stage that converts 1.0 models to 1.0.1.

We also want to separate `addDefaults` into `addDefaults` and `convert 0.8 to `1.0`.  Right now it does both at the same time.

See #1 and https://github.com/KhronosGroup/glTF/wiki/glTF-0.8-to-1.0-Guide.

You are welcome to take a look at this now if you want.